src/Store.cs

```diff
@@ -61,7 +61,12 @@ internal void SetWasiConfiguration(WasiConfiguration config)
             }
         }
 
-        private static class Native
+        public void SetEpochDeadline(ulong deadline)
+        {
+            Native.wasmtime_context_set_epoch_deadline(handle, deadline);
+        }
+
+        public static class Native
```

Comment:
I think this inner class should probably remain `private` or at the very least `internal`.